modules/viewport-mercator-project/src/web-mercator-utils.js

```diff
@@ -59,10 +58,9 @@ export function lngLatToWorld([lng, lat], scale) {
  *   Has toArray method if you need a GeoJSON Array.
  *   Per cartographic tradition, lat and lon are specified as degrees.
  */
-export function worldToLngLat([x, y], scale) {
-  scale *= TILE_SIZE;
-  const lambda2 = (x / scale) * (2 * PI) - PI;
-  const phi2 = 2 * (Math.atan(Math.exp(PI - (y / scale) * (2 * PI))) - PI_4);
+export function worldToLngLat([x, y]) {
```

Comment:
Update docs for this change? and Upgrade guide?

Reply:
This is a new module.